components/admin_console/custom_terms_of_service_settings/custom_terms_of_service_settings.jsx

```diff
@@ -0,0 +1,235 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+import PropTypes from 'prop-types';
+
+import {saveConfig} from 'actions/admin_actions.jsx';
+import AdminSettings from 'components/admin_console/admin_settings.jsx';
+
+import SettingsGroup from 'components/admin_console/settings_group.jsx';
+import BooleanSetting from 'components/admin_console/boolean_setting.jsx';
+import TextSetting from 'components/admin_console/text_setting.jsx';
+import FormattedMarkdownMessage from 'components/formatted_markdown_message.jsx';
+import LoadingScreen from 'components/loading_screen.jsx';
+
+import {Constants} from 'utils/constants';
+
+export default class CustomTermsOfServiceSettings extends AdminSettings {
```

Comment:
Why `Custom`? Of course it's custom, it's a setting.

I guess this is already set in stone because of the config setting though...

Reply:
Correct, and it follows how other apps name this setting.